frontend/src/styles/colors.sass

```diff
@@ -2,6 +2,9 @@ $pink-100: #F8D5E8
 $white: #FFFFFF
 $turquoise-100: #61D4F6
 $blue-200: #1C2FBA
+$blue-100: #E0E2F2
```

Comment:
We have this color, so either change everywhere either use existing

Reply:
My bad, now i will check if color already exist